lib/fishjam/rpc_client.ex

```diff
@@ -48,6 +48,7 @@ defmodule Fishjam.RPCClient do
 
         acc
     end)
+    |> List.flatten()
```

Comment:
Why do we need that?

Reply:
When listing rooms, every node returns a list. Without this call, the result was a nested list of lists.

Reply:
Okay, but this is valid only in the case of listing a room; in other cases, it doesn't have to, so I think this is not a good place to use it.